lib/main-staging.dart

```diff
@@ -0,0 +1,7 @@
+import 'flavors.dart';
```

Comment:
@luongvo This file is redundant because we don't use `flutter-flavorizr` anymore 🙋